### PR TITLE
feat: update tests

### LINKED ISSUE #323: Update tests

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Fixes #323

### CODE REVIEW

**Review by Rubilmax — COMMENTED** [review]
Can we base this PR on main?

**Comment by Rubilmax** [comment]
Because cantina review is outdated ^^ We merged it yesterday

**Comment by Jean-Grimal** [comment]
> Because cantina review is outdated ^^ We merged it yesterday

Oh yes mb

File: test/forge/ReallocateWithdrawTest.sol [hunk 1]
```diff
@@ -104,6 +115,36 @@ contract ReallocateWithdrawTest is IntegrationTest {
...
     }
 
+    function testReallocateWithdrawIncreaseSupply(uint256[3] memory assets) public {
+        _setCap(allMarkets[2], 3 * CAP2);
+
+        allocations.push(MarketAllocation(allMarkets[0], 0));
+        allocations.push(MarketAllocation(allMarkets[1], 0));
+        allocations.push(MarketAllocation(allMarkets[2], 2 * CAP2));
+
+        vm.startPrank(ALLOCATOR);
```
Comment (Rubilmax) [inline test/forge/ReallocateWithdrawTest.sol:125]: Ideally we only prank the `reallocate` call otherwise this PR is neat!

**Review by MerlinEgalite — APPROVED** [review]
Nice thx!

File: test/forge/ReallocateWithdrawTest.sol (hunk 1 above)
Comment (Rubilmax) [reply test/forge/ReallocateWithdrawTest.sol:148]: Did you move it right before `reallocate`?

Comment (Jean-Grimal) [reply test/forge/ReallocateWithdrawTest.sol]: Fixed. ty

**Pull request merged**